projects/core/src/occ/adapters/checkout/default-occ-checkout-config.ts

```diff
@@ -4,9 +4,27 @@ export const defaultOccCheckoutConfig: OccConfig = {
   backend: {
     occ: {
       endpoints: {
+        cardTypes: 'cardtypes',
+        paymentTypes: 'paymenttypes',
+        /* eslint-disable max-len */
         setDeliveryAddress:
           'users/${userId}/carts/${cartId}/addresses/delivery',
+        deliveryAddresses: 'users/${userId}/carts/${cartId}/addresses/delivery',
+        deliveryMode: 'users/${userId}/carts/${cartId}/deliverymode',
+        deliveryModes: 'users/${userId}/carts/${cartId}/deliverymodes',
         placeOrder: 'users/${userId}/orders?fields=FULL',
+        cartCostCenter:
```

Comment:
Can we name this `setCartCostCenter` ?
```suggestion
        setCartCostCenter:
```